model/src/main/kotlin/config/CliConfiguration.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+/**
+ * The configuration model of the CLI.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class CliConfiguration(
```

Comment:
Why did you choose `cli` instead of `helper-cli`?

Reply:
I just talked to @mnonnenmacher also about that, and one of the reasons is that helper-cli does not even have the option to read a HOCON config file yet.